docs/io.cozy.procedures.md

```diff
@@ -0,0 +1,31 @@
+[Table of contents](README.md#table-of-contents)
+
+# Cozy Procedures doctype
+
+## `io.cozy.procedures`
+
+The `io.cozy.procedures` doctype represents the administrative procedures.
+
+- `templateId`: {string} the id of the `io.cozy.procedures.templates` the procedure is related to.
+- `data`: {object} the data about the procedure (_e.g_ amount, duration, _etc_).
+- `personalData`: {object} the personal data associated to the procedure (_e.g_ email, phone number, _etc_).
+- `relationships`: {object} links between documents
+  - `files`: {object} attachments
```

Comment:
How do we know at what this file is linked? Should we have an id somewhere in the template and store it here?  (even if we don't create a doctype yet?) https://github.com/cozy/cozy-doctypes/pull/93/files#diff-133965ee0369135052637dfc7cbbb86cR25